corehq/apps/sms/models.py

```diff
@@ -2297,3 +2290,6 @@ def has_migration_completed(cls, name):
             return True
         except cls.DoesNotExist:
             return False
+
+
+from corehq.apps.sms.signals import *
```

Comment:
does this have to be `*`? this fails the grep test and i know was a source of confusion when dealing with the `User` model